roles/postgresql/tasks/postgresql_yum6.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: yum |  Setup PostgreSQL repository for {{ ansible_distribution }}
+  become: yes
+  yum:
+    name: https://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-centos94-9.4-3.noarch.rpm
+    state: installed
+  when: ansible_distribution == "CentOS"
+  tags: [db, postgresql]
+
+- name: yum | Setup PostgreSQL repository for {{ ansible_distribution }}
+  become: yes
+  yum:
+    name: https://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-redhat94-9.4-3.noarch.rpm
+    state: installed
+  when: ansible_distribution == "RedHat"
+  tags: [db, postgresql]
+
+- name: yum | Install postgresql dependency
+  become: yes
+  yum:
+    name: libselinux-python
+    state: installed
+  tags: [db, postgresql]
+
+- name: yum | Install postgresql94
+  become: yes
+  yum:
+    name: "{{ item }}"
+    state: installed
+    update_cache: yes
```

Comment:
Can we avoid `update_cache` here to make this command Ansible-idempotent friendly?

With update_cache it will always have "changed" state.